### PR TITLE
add 'abbr_number' and fix the position of the rank tier tooltip

### LINKED ISSUE #1383: replace "No. " string
https://github.com/odota/web/blob/2d252ebaaa9c54f1a6813f9bbc38bbf872c172c9/src/components/Player/Header/PlayerHeader.jsx#L166

@GravityMsc could you replace "No." with a `#` symbol or move it to the string file?

### PR DESCRIPTION
**Pull request by GravityMsc**
fix #1383

### CODE REVIEW

File: src/lang/en-US.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "abbr_not_available": "N/A",
   "abbr_pick": "P",
   "abbr_win": "W",
+  "abbr_number": "#",
```
Comment (howardchung) [inline src/lang/en-US.json]: if you're using a `#` symbol i don't think you need to use the string file since it doesn't need to be translated.

Comment (GravityMsc) [reply src/lang/en-US.json]: I think this is necessary. I want to use '排名' in Chinese file.

Comment (GravityMsc) [reply src/lang/en-US.json]: and to be honest, I don't know what is the proper word of the U.S.

Comment (howardchung) [reply src/lang/en-US.json]: `No.` is ok for English.  If you use `#` in the string file the translators probably won't know what to do with it.

Comment (GravityMsc) [reply src/lang/en-US.json]: I used 'No.' as a temporary word at the beginning, lol. I misunderstood it.

**Pull request merged**